CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 **Note**: If the strings translations change it will result in a MINOR version change, therefore you are responsible for testing your translated layout in case you are using custom translations. [More on language localisation](README.md#language-customisation)
 
+## [22.2.0] - 2021-08-09
```

Comment:
Why using the square brackets if these are not links? On the Web SDK we use square brackets for versions because at the bottom of the file there is a link where you can check the diff between master and development for each version, but it doesn't make sense to have them here IMO.